.github/workflows/release-process.yml

```diff
@@ -104,8 +105,9 @@ jobs:
 
       - name: Perform latest release
         run: |
+          set -ex
           git fetch origin
-          cur=$(git for-each-ref --sort=-committerdate refs/remotes/origin --format '%(refname)' | grep release | head -n 1 | sed 's/.*release-//')
+          cur=$(git for-each-ref refs/remotes/origin --format '%(refname)' | grep release | sed 's/.*release-//' | sort -n -r | head -n 1)
```

Comment:
Unfortunately I don't think the `-n` on sort does what we want here, if we're dealing with semvers rather than plain integers. I just tested with an input of

```
0.1.0
0.10.0
0.2.0
```

and `sort -n -r` on that gives `0.2.0`, `0.10.0`, `0.1.0` for me.

We have all two-digit versions *for now* so it will kind of work but this will bite us if/when we hit three-digit or go back to one-digit versions...

I know this is a big ask, but the complexity here is starting to get to the point where I might wonder if it would be better to write a small Rust script that understands versions, and is more resilient than `sed` commands on magic strings?

Alternately I don't think it's a *huge* deal to ask a human to edit the date in a final PR, compared to the effort and risk that goes into automating the last mile here...